azure-pipelines.yml

```diff
@@ -11,7 +11,7 @@ variables:
 - name: 'windows_firmware_version'
   value: '1.6.102075014'
 - name: 'PackageVersion'
-  value: '1.2.0-alpha.10'
+  value: '1.2.0'
```

Comment:
Should this be 1.3.0-alpha.0?

Reply:
No, this is the NuGet package name to download and get the depth engine from